src/overlays/actors/ovl_En_Daiku2/z_en_daiku2.h

```diff
@@ -5,13 +5,50 @@
 
 struct EnDaiku2;
 
-typedef void (*EnDaiku2ActionFunc)(struct EnDaiku2* this, GlobalContext* globalCtx);
+typedef void (*EnDaiku2ActionFunc)(struct EnDaiku2*, GlobalContext*);
+
+#define ENDAIKU2_GET_7F(thisx) ((thisx)->params & 0x7F);
+#define ENDAIKU2_GET_1F80(thisx) (((thisx)->params >> 7) & 0x3F);
+#define ENDAIKU2_GET_8000(thisx) (((thisx)->params >> 15) & 0x1);
```

Comment:
```suggestion
#define ENDAIKU2_GET_PARAMS_7F(thisx) ((thisx)->params & 0x7F);
#define ENDAIKU2_GET_PARAMS_1F80(thisx) (((thisx)->params >> 7) & 0x3F);
#define ENDAIKU2_GET_PARAMS_8000(thisx) (((thisx)->params >> 15) & 0x1);
```
Similar to other review comments, `GET_8000` is very ambiguous and could mean a lot of things

Reply:
He doesn't know what they do. The names are placeholders for now. I think it's fine. They'll be renamed when documentation is done on the file.